go.mod

```diff
@@ -70,6 +70,6 @@ require (
 	golang.org/x/net v0.0.0-20220811182439-13a9a731de15 // indirect
 	golang.org/x/sync v0.0.0-20220601150217-0de741cfad7f // indirect
 	golang.org/x/sys v0.0.0-20220811171246-fbc7d0a398ab // indirect
-	golang.org/x/text v0.3.8 // indirect
+	golang.org/x/text v0.4.0 // indirect
```

Comment:
This vulnerability had already been fixed in acf83e743a23df0bdba8e7565bf496cf6aa2b039
Previous version is 0.3.8 which is not vulnerable, but this update should still be ok

Reply:
True, master pipeline was broken from a release i just merged; however the release branch did not contain the fix on x/text vulnerability that can be seen in develop branch (I didn't see this) and just upgraded the dependency.

I will merge this and create a release cut, so master pipeline deploys secauth changes